pkg/rtc/transport.go

```diff
@@ -1563,7 +1606,10 @@ func (t *PCTransport) setRemoteDescription(sd webrtc.SessionDescription) error {
 	}
 
 	for _, c := range t.pendingRemoteCandidates {
-		if err := t.pc.AddICECandidate(*c); err != nil {
+		err := t.handleRemoteICECandidate(&event{
```

Comment:
@boks1971 I noticed that the pending candidates wouldn't be filtered nor added to remote candidate list. addressing it here. does it seem right?

Reply:
@davidzhao No, this change is not needed. Don't need to go through `handleRemoteICECandidate` again. It would have gone through that path when the ICE candidate is received.